shared/templates/audit_rules_privileged_commands/template.py

```diff
@@ -2,10 +2,24 @@
 import os
 
 
+def default_boolean(data, parameter, default):
+    if parameter not in data:
+        data[parameter] = default
+    if data[parameter].strip().lower() == "true":
+        data[parameter] = True
+    else:
+        data[parameter] = False
+
+
 def preprocess(data, lang):
     path = data["path"]
     name = ssg.utils.escape_id(os.path.basename(path))
     data["name"] = name
+    if "min_auid" not in data:
+        data["min_auid"] = 1000
+    default_boolean(data, "skip_action", "false")
+    default_boolean(data, "watch", "false")
+
```

Comment:
```suggestion
from ssg.utils import parse_template_boolean_value


def preprocess(data, lang):
    path = data["path"]
    name = ssg.utils.escape_id(os.path.basename(path))
    data["name"] = name
    if "min_auid" not in data:
        data["min_auid"] = 1000
    parse_template_boolean_value(data, parameter="skip_action", default_value=False)
    parse_template_boolean_value(data, parameter="watch", default_value=False)

```

Reply:
Then add these two new parameters to the documentation of this template, here: 

https://github.com/ComplianceAsCode/content/blame/master/docs/templates/template_reference.md#L80